soc/arc/snps_nsim/Kconfig

```diff
@@ -29,6 +29,11 @@ config SOC_NSIM_HS_SMP
 	bool "Multi-core Synopsys ARC HS in nSIM"
 	select CPU_HAS_FPU
 
+config SOC_NSIM_HS_MPUV6
+	bool "Synopsys ARC HS with MPU v6 in nSIM"
+	select CPU_HAS_MPU
+	select CPU_HAS_FPU
```

Comment:
Why is FPU enabled?

Reply:
I keep it same with other NSIM_HS boards, so FPU is enabled